content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,188 @@
+- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
+  with OCR and AI.
+- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
+- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
+- :guilabel:`Lead Generation`: generates leads based on a set of criteria, and converts web visitors
+  into quality leads and opportunities.
+- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.
 
-IAP Portal
-==========
+For more information on every service currently available, visit the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+Use IAP services
+----------------
 
-.. image:: in_app_purchase/image3.png
-   :align: center
+IAP services are automatically integrated with Odoo, and do **not** require users to configure any
+settings. To use a service, simply interact with it wherever it appears in the database.
 
-Get notified when credits are low
-=================================
+.. example::
+   The following flow focuses on the *SMS* IAP service being used from a contact's record.
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: The SMS icon on a typical contact information form located within an Odoo database.
+
+   One way to utilize the *SMS* IAP service with Odoo is showcased in the following steps:
+
+   1. Navigate to the *Contacts* application, and click on a contact with a mobile phone number
+      entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the contact form.
+   2. Hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
+      (phone) SMS` icon appears to the right.
+   3. Click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text Message` pop-up
+      window appears.
+   4. Type a message in the :guilabel:`Message` field of the :guilabel:`Send SMS Text Message`
+      pop-up window. Then, click the :guilabel:`Send SMS` button. Odoo then sends the message, via
+      SMS, to the contact, and logs what was sent in the *chatter* of the contact's form.
```

Comment:
You can use the dynamic `#.` instead of static `1.` so that the number of bullets is automatically incremented, and is [proffered](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#numbered-list) 
```suggestion
   #. Navigate to the *Contacts* application, and click on a contact with a mobile phone number
      entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the contact form.
   #. Hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
      (phone) SMS` icon appears to the right.
   #. Click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text Message` pop-up
      window appears.
   #. Type a message in the :guilabel:`Message` field of the :guilabel:`Send SMS Text Message`
      pop-up window. Then, click the :guilabel:`Send SMS` button. Odoo then sends the message, via
      SMS, to the contact, and logs what was sent in the *chatter* of the contact's form.
```